### PR TITLE
[7.67.x-blue][BXMSPROD-2057] Updating references to the new kie-ci actions folder

### PR DESCRIPTION
**Pull request by bacciotti**

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -24,18 +24,19 @@ jobs:
...
+        uses: kiegroup/kie-ci/.ci/actions/maven@main
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/build-chain@main
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@main
         with:
+          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/ pull-request-config.yaml
```
Comment (lampajr) [inline .github/workflows/pull_request.yml]: as we did for `7.67.x`
```suggestion
          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/droolsjbpm-build-bootstrap/${BRANCH:main}/.ci/pull-request-config.yaml
```

**Review by lampajr — CHANGES_REQUESTED** [review]

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (mareknovotny) [reply .github/workflows/pull_request.yml]: i fixed redundant space after `.ci` in your suggestion @lampajr

Comment (lampajr) [reply .github/workflows/pull_request.yml]: oh thanks Marek, missed that

**Review by lampajr — APPROVED** [review]

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**